jest.config.js

```diff
@@ -1,14 +1,15 @@
 module.exports = {
   rootDir: 'spec',
   transformIgnorePatterns: [
-    'node_modules/(?!(@18f/identity-address-search)/)',
+    'node_modules/(?!(@18f/identity-address-search|@18f/identity-i18n)/)',
     '\\.pnp\\.[^\\/]+$',
   ],
   moduleNameMapper: {
     '@18f/identity-address-search':
       '<rootDir>/../node_modules/@18f/identity-address-search/dist/index.js',
+    '@18f/identity-i18n': '<rootDir>/../node_modules/@18f/identity-i18n/dist/index.js',
   },
-  testMatch: ['**/*_spec.[jt]s'],
+  testMatch: ['**/*_spec.[jt]s', '**/*_spec.[jt]s?'],
```

Comment:
This would accept file extensions `*.t` and `*.j`, which I doubt is what you want. Usually it's more like `**/*_spec.[jt]sx?` to accept files that use [JSX](https://www.typescriptlang.org/docs/handbook/jsx.html) in addition to JavaScript and TypeScript files.

```suggestion
  testMatch: ['**/*_spec.[jt]s'],
```

Reply:
@NavaTim  When I change the testMatch to the above, no tests are found. Inside the Summary of Changes in my description I mentioned a few textMatchers I tried. This is the only combo I found to get my .tsx test file to run in addition to all of the others. Reach out on Slack if you'd like to jump on a call. 

<img width="935" alt="Screenshot 2023-09-15 at 1 58 37 PM" src="https://github.com/18F/identity-site/assets/125507397/63db5ff4-9b6a-4bc8-8983-44b4c70e5d97">

Reply:
The Jest docs for [`testMatch`](https://jestjs.io/docs/configuration#testmatch-arraystring) says that it uses [micromatch](https://github.com/micromatch/micromatch), which works a little differently than other pattern-matching systems I'm familiar with.

I experimented and found that using [the extglob version](https://github.com/micromatch/micromatch#extended-globbing) of what I was trying to do (which is what Jest's default pattern does) seems to get the job done.

```suggestion
  testMatch: ['**/*_spec.[jt]s?(x)'],
```

Reply:
👏  You got it! I feel like I tried half a dozen different versions but apparently not that one.